pkg/common/test/recoverable_apps_mock.go

```diff
@@ -22,7 +22,6 @@ import (
 	"github.com/apache/yunikorn-k8shim/pkg/common/constants"
 	"k8s.io/api/core/v1"
 
-	"github.com/apache/yunikorn-k8shim/pkg/appmgmt/interfaces"
 	"github.com/apache/yunikorn-scheduler-interface/lib/go/si"
```

Comment:
nit: sorting imports, first group system import, second group other imports, third group yunikorn imports
can be fixed on commit